guides/source/routing.md

```diff
@@ -1386,15 +1439,31 @@ URI               | /users/:id/edit(.:format)
 Controller#Action | users#edit
 ```
 
-You can search through your routes with the grep option: -g. This outputs any routes that partially match the URL helper method name, the HTTP verb, or the URL path.
+### Routes in Rails Console
+
+You can access route helpers using `Rails.application.routes.url_helpers` within the [Rails Console](command_line.html#bin-rails-console). For example:
+
+```irb
+irb> app.users_path
+=> "/users"
+
+irb> user = User.first
+=> #<User:0x00007fc1eab81628
+irb> app.edit_user_path(user)
+=> "/users/1/edit"
+```
```

Comment:
I find it confusing to state that you can use helpers via `` when the example uses `app.url_helper` directly.

```suggestion
You can access route helpers using `Rails.application.routes.url_helpers` within the [Rails Console](command_line.html#bin-rails-console). They are also available via the [app](command_line.html#the-app-and-helper-objects) object. For example:

```irb
irb> Rails.application.routes.url_helpers.users_path
=> "/users"

irb> user = User.first
=> #<User:0x00007fc1eab81628
irb> app.edit_user_path(user)
=> "/users/1/edit"
```
```

Reply:
That's a good clarification, thanks!